main.py

```diff
@@ -132,56 +179,105 @@ def unp(self):
+    def to_dict(self):
+        return {
+            "client_id": self._client_id,
+            "creation_date": self._creation_date,
+            "unp": self._unp,
+            "name": self._name,
+            "role": self._role.to_dict,
+        }
 
-class App(Entity):
-    def __init__(self, entity_id: int, role, name):
-        super().__init__(entity_id, role)
+    @property
+    def to_dict_write(self):
+        return {
+            "client_id": self._client_id,
+            "creation_date": self._creation_date,
+            "unp": self._unp,
+            "name": self._name,
+            "role": self._role.name,
+        }
+
+class App(Client):
+    def __init__(self, client_id: int, role, name):
+        super().__init__(client_id, role)
         self._name = name
 
     @property
     def name(self):
         return self._name
 
 
-# функция добавления пользователя
-def create_user(role, first_name, last_name, fathers_name, date_of_birth):
-    entity_id = array_users[-1].entity_id + 1
+# add new user
+def create_user(data):
+    client_id = clients[-1].client_id + 1
     new_user = User(
-        entity_id, array_roles[role], first_name, last_name, fathers_name, date_of_birth
+        client_id,
+        roles[data["role"]],
+        data["first_name"],
+        data["last_name"],
+        data["fathers_name"],
+        int(data["date_of_birth"]),
+    )
+    clients.append(new_user)
+
+
+# add new organisation
+def create_organisation(data):
+    client_id = clients[-1].client_id + 1
+    new_user = Organisation(
+        client_id,
+        roles[data["role"]],
+        int(data["creation_date"]),
+        int(data["unp"]),
+        data["name"],
     )
-    array_users.append(new_user)
+    clients.append(new_user)
+
+
+# write to json
+def write_json():
+    write_clients = {"Users": [], "Organisations": []}
+    for client in clients:
+        if isinstance(client, User):
+            write_clients["Users"].append(client.to_dict_write)
+        else:
+            write_clients["Organisations"].append(client.to_dict_write)
+    with open("users.json", "w") as f:
+        json.dump(write_clients, f, ensure_ascii=False)
 
 
 def main():
     # создаем пустые массивы
-    global array_users
-    global array_roles
-    array_users = []
-    array_apps = []
-    array_roles = {}
+    global clients
+    global roles
+    clients = []
+    apps = []
+    roles = {}
 
     # получаем список roles
     with open("roles.yaml", "r") as f:
         roles_yaml = yaml.safe_load(f)
     for key in roles_yaml.keys():
-        array_roles[key] = Role(key, roles_yaml[key])
+        roles[key] = Role(key, roles_yaml[key])
 
-    # получаем список app
+    # получаем список apps
     with open("app.yaml", "r") as f:
         app_yaml = yaml.safe_load(f)
 
     for app in app_yaml["Apps"]:
-        array_apps.append(App(app["entity_id"], app["role"], app["name"]))
+        apps.append(App(app["client_id"], roles[app["role"]], app["name"]))
```

Comment:
Тут тоже надо 
```suggestion
        clients.append(App(app["client_id"], roles[app["role"]], app["name"]))
```
Тк наши сервисы такие как банк, аккаунт, тоже являются клиентами authz сервиса

Reply:
готово